packages/manager/modules/pci/src/projects/project/training/jobs/resubmit/resubmit.html

```diff
@@ -0,0 +1,45 @@
+<form
+    id="trainingResubmitJob"
+    name="trainingResubmitJob"
+    data-ng-submit="$ctrl.confirmResubmitJob()"
+>
+    <oui-modal
+        data-primary-label="{{:: 'pci_projects_project_training_jobs_resubmit' | translate }}"
+        data-secondary-label="{{:: 'pci_projects_project_training_jobs_common_cancel' | translate }}"
+        data-secondary-action="$ctrl.goBack()"
+        data-on-dismiss="$ctrl.goBack()"
+        data-loading="$ctrl.loading"
+    >
+        <oui-tile
+            class="h-100"
+            data-heading="{{:: 'pci_projects_project_training_jobs_confirm_resubmit' | translate  }}"
+        >
+            <oui-tile-definition
+                data-term="{{:: 'pci_projects_project_training_jobs_id' | translate  }}"
+            >
+                <oui-tile-description
+                    data-ng-bind="$ctrl.job.id"
+                ></oui-tile-description>
+            </oui-tile-definition>
+
+            <oui-tile-definition
+                data-term="{{:: 'pci_projects_project_training_jobs_region' | translate  }}"
+            >
+                <oui-tile-description
+                    data-ng-bind="$ctrl.job.spec.region"
+                ></oui-tile-description>
+            </oui-tile-definition>
+            <oui-tile-definition
+                data-term="{{:: 'pci_projects_project_training_jobs_submit_validation_equivalent_command' | translate  }}"
+            >
+                <oui-tile-description>
+                    <span
+                        data-translate="pci_projects_project_training_jobs_submit_ovhai_info"
+                    ></span>
+                    <pre class="ng-scope">
+<code class="ng-binding">{{:: 'pci_projects_project_training_parter_cli_name' | translate }} {{ $ctrl.getResubmitCliCommand() }} </code></pre>
```

Comment:
Following `ng-binding` class can be removed (and remove extra white space).

```suggestion
<code>{{:: 'pci_projects_project_training_parter_cli_name' | translate }} {{ $ctrl.getResubmitCliCommand() }}</code></pre>
```